js/menu.js

```diff
@@ -20,3 +20,169 @@ hamburgerIcon.addEventListener('click', appear);
+    seeBtn: 'See Project',
+  },
+  {
+    id: 'card4',
+    name: 'Uber Navigation',
+    description: 'A daily selection of privately personalized reads, no accounts or sign-ups required.',
+    imageLink: './img/Snapshoot-Portfolio-4.png',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    role: 'Lead Developer',
+    company: 'UBER',
+    year: '2018',
+    sourceLink: 'https://github.com/IvonneBenitesRodriguez/microverse_mobile_portfolio',
+    demoLink: 'https://ivonnebenitesrodriguez.github.io/microverse_mobile_portfolio/',
+    icon: 'Counter.svg',
+    seeBtn: 'See Project',
+  },
+];
+
+workSections.innerHTML = '';
+for (let i = 0; i < projectsData.length; i += 1) {
+  // let languages = '';
+  // projectsData[i].technologies.forEach((language) => {
+  //   // workSections.innerHTML += `
+  // // <li><a href='' class='skill' title='${language}'>${language}</a></li>
+  // // `;
+  // });
+
+  workSections.innerHTML += `
+          <div class="card-works'>
+                        <figure class="desktop-first">
+                          <img src='${projectsData[i].imageLink}' alt='work 1' class='' />
+                        </figure>
+                <div class='left-block'>
+                          <h2 class='primary-text'>${projectsData[i].name}</h2>
+                          <div class='list-card-primary'>
+                                <ul>
+                                  <li><a href="#" class="canopy-card-1">${projectsData[i].company}</a></li>
+                                  <li><a href="#" class="back">${projectsData[i].role}</a></li>
+                                  <li><a href="#" class="year">${projectsData[i].year}</a></li>
+                                </ul>
+                          </div>      
+                      <p class="work-text">${projectsData[i].description}</p>
+                      <div class="buttons-group">
+                          <ul class="buttons">
+                              <li><button class="button-bg"><a href="#">${projectsData[i].technologies[0]}</a></button></li>
+                              <li><button class="button-bg"><a href="#">${projectsData[i].technologies[1]}</a></button></li>
+                              <li><button class="button-bg"><a href="#">${projectsData[i].technologies[2]}</a></button></li>
+                          </ul>
+                      </div>
+                     <button type='button' class='normal-btn btn btn-enabled project-modal-${i}'>See project</a></button>
+                  </div>
+              </div>
+             
+     `;
+}
+
+for (let i = 0; i < projectsData.length; i += 1) {
+  document.querySelector(`.project-modal-${i}`).addEventListener('click', () => {
+    projectName.innerHTML = projectsData[i].name;
+    modalImg.src = projectsData[i].imageLink;
+    company.innerHTML = projectsData[i].company;
+    role.innerHTML = projectsData[i].role;
+    year.innerHTML = projectsData[i].year;
+
+    modal.innerHTML = `
```

Comment:
- [ ] On the desktop view, I can't scroll down to see the modal information and buttons, kindly make sure to fix this so the modal fits desktop sizes properly

![Screenshot_20230729_162816](https://github.com/IvonneBenitesRodriguez/microverse_mobile_portfolio/assets/25227022/3dd2a173-175c-4f7e-bf91-77061c3e107e)